cmd/summary.go

```diff
@@ -233,17 +155,91 @@ func GetSummaryCommand() *cobra.Command {
 						}
 						return errors.Join(errs...)
 					}
-					return nil
 				}
+			} else {
+				// if an invalid number of arguments are provided, print usage
+				PrintHowToUse("summary")
+				return errors.New("invalid number of arguments provided")
```

Comment:
Should we just return an error, or should I also do a `pterm.Error.println(...)`?